scripts/apps/ingest/services/IngestProviderService.js

```diff
@@ -16,6 +16,36 @@ export function IngestProviderService(api, $q, preferencesService, $filter, sear
             });
     };
 
+    var _getAllFeedParsersAllowed = function(criteria = {}, page = 1, parsers = []) {
+        return api.query('feed_parsers_allowed', _.extend({max_results: 200, page: page}, criteria))
+            .then((result) => {
+                let pg = page;
+                let merged = parsers.concat(result._items);
+
+                if (result._links.next) {
+                    pg++;
+                    return _getAllFeedParsersAllowed(criteria, pg, merged);
+                }
+                return $filter('sortByLabel')(merged);
+            });
+    };
+
+    // TODO: not implemented yet.  should be used when dynamic services with custom
```

Comment:
It's a bit unclear what is not implemented... The function seems to have code, so it is implemented. Or do you mean the API endpoint? If so, maybe the comment belongs above the line `api.query` and might be worth specifying that the backend is unimplemented. And if that's the case, then maybe it's not even worth having this comment here?